docs/se/dbclient.adoc

```diff
@@ -24,6 +24,16 @@
 
 include::{rootdir}/includes/se.adoc[]
 
+==Contents
```

Comment:
I don't know if either AsciiDoctor or our tool chain (or neither) requires a space between the `==` and the header text. Maybe double-check that this renders properly with a local build?

Reply:
A typo on my part.